Conceptpower+Spring/src/main/java/edu/asu/conceptpower/servlet/rest/ConceptIDLookup.java

```diff
@@ -84,6 +85,12 @@
 
         if (entry != null) {
 
+            try {
```

Comment:
The returned concept will still have the specific Wordnet id in the id field, in cases where we query for the general Wordnet id, won't it? Since you never set the id field (or do you?), and the manager returns a wordnet concept (not the concept wrapping a wordnet) in cases where the wordnet id used in the query is a general wordnet id, but the one used for wrapping is a specific one. Does this make sense?